actionpack/test/controller/rescue_test.rb

```diff
@@ -314,15 +286,14 @@ def test_block_rescue_handler_with_argument
     get :resource_unavailable
     assert_equal "RescueController::ResourceUnavailable", @response.body
   end
-
   def test_block_rescue_handler_with_argument_as_string
     get :resource_unavailable_raise_as_string
     assert_equal "RescueController::ResourceUnavailableToRescueAsString", @response.body
   end
 
   test 'rescue when wrapper has more specific handler than cause' do
     get :exception_with_more_specific_handler_for_wrapper
-    assert_response :unprocessable_entity
+    assert_response :forbidden
```

Comment:
Exception handler always has priority now. If no handler matches, we use the cause's handler.